src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(getRandomBall());
+        }
     }
 }
```

Comment:
Classes should have access modifiers. In this case, `Application` should be declared as `public`.